### DISCUSSION TITLE
关于舵机调试遇到的问题

### DISCUSSION THREAD

**Discussion by GreenShadeZhang** [peng-zhihui/ElectronBot#178] [Q&A]
本人通过采用源码里提供的舵机调试工具在win11的系统上进行调试，出现软件显示disconnected的问题。
![image](https://user-images.githubusercontent.com/24240675/168122675-ed90da04-a639-4254-aaaf-f390434d2485.png)

在用stkink unity烧录f042的固件时失败，但是烧录主控正常，有人建议可能是软件版本过低。
在clion通过编译舵机驱动源码确实可以进行烧录，结果如下图。
![image](https://user-images.githubusercontent.com/24240675/168121292-875ce090-a092-43bb-9be2-36892d438847.png)
但是采用问题讨论区的代码进行舵机的往复运动操作时没有反应，怀疑是焊接有问题，但是测试了五个舵机都不行。
可能存在舵机驱动烧录没问题，但是焊接有问题的情况吗？
比如我要检测电容电阻还是什么。因为我用主板的舵机插槽也是没反应排除可能传感器板子的问题。
现在我是不太好定位问题具体出在哪里，大家有没有什么思路。

**Comment by Kitem** [comment]
您好，我是许焕挺，已收到您的来信，谢谢。
